doc/python/marker-style.md

```diff
@@ -305,6 +305,115 @@ fig.show()
 
 ```
 
+### Custom Marker Symbols
+
+The `marker_symbol` attribute allows you to choose from a wide array of symbols to represent markers in your figures.
+
+The basic symbols are: `circle`, `square`, `diamond`, `cross`, `x`, `triangle`, `pentagon`, `hexagram`, `star`, `diamond`, `hourglass`, `bowtie`, `asterisk`, `hash`, `y`, and `line`. 
+
+Each basic symbol is also represented by a number. Adding 100 to that number is equivalent to appending the suffix "-open" to a symbol name. Adding 200 is equivalent to appending "-dot" to a symbol name. Adding 300 is equivalent to appending "-open-dot" or "dot-open" to a symbol name.
+
+In the following figures, hover over a symbol to see its name or number. Set the `marker_symbol` attribute equal to that name or number to change the marker symbol in your figure.
+
+#### Basic Symbols
+
+```python
+import plotly.graph_objects as go
+fig = go.Figure()
+fig.update_layout(title="Basic Symbols")
+fig.update_xaxes(showticklabels=False)
+fig.update_yaxes(showticklabels=False)
+
+for index in range(27):
+    fig.add_trace(go.Scatter(x=[(index % 30)], y=[index // 30], 
+                             marker_symbol=index, marker_color='black', 
+                             marker_size=10, showlegend=False, hovertext=index))
```

Comment:
If you're not tired of me nitpicking you could use a hovertemplate here to show just the index, not the coordinates.